sprocs/index.js

```diff
@@ -29,134 +29,135 @@ module.exports = {
     _initWithLock(callback) {
         logger.verbose('Starting DB stored procedure initialization');
         async.eachSeries([
-            'array_dot.sql',
```

Comment:
Don't alphabetize this list. The load order is important for sprocs that reference other sprocs. (If you were to load PL on an empty database doing the loads, you'd see it crashes loading sprocs.) I'd suggest reverting your changes and adding your sprocs at the end of the file.

These changes are also causing a merge conflict with master, which is breaking the CI testing. You'll want to make sure you can merge `master` into your branch without challenges?